tools/update_certs.py

```diff
@@ -0,0 +1,151 @@
+#!/usr/bin/python3
+# Copyright 2024 The Cobalt Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Tool for updating Cobalt SSL Certificates.
```

Comment:
Overall this is neat and clean, good component. Would you mind adding `update_certs_test.py` and cover SplitCertSet, ComputeCertFilename and UpdateBuild functions with unit tests ?

Reply:
will do!